synthetic_data/synthetic_data.py

```diff
@@ -305,9 +306,112 @@ def make_tabular_data(
     #
 
     if noise_level_x > 0.0:
-        x_noise = generate_x_noise(
-            x_final[:, :n_informative], noise_level_x, seed=seed
-        )
+        x_noise = generate_x_noise(x_final[:, :n_informative], noise_level_x, seed=seed)
```

Comment:
Per PEP8, I think you actually want to leave this on multi-lines... the new change to put it all on one line is a len of 89.